src/gpu/OpenCL.cpp

```diff
@@ -24,12 +24,14 @@
 
 
 static iware::gpu::vendor_t parse_vendor(const char* name) {
-	if(!std::strcmp(name, "Intel(R) Corporation"))
+	if(!std::strcmp(name, "Intel(R) Corporation") || !std::strcmp(name, "Intel"))
 		return iware::gpu::vendor_t::intel;
-	else if(!std::strcmp(name, "Advanced Micro Devices, Inc."))
+	else if(!std::strcmp(name, "Advanced Micro Devices, Inc.") || !std::strcmp(name, "AMD"))
```

Comment:
Do these actually fall out of `clGetDeviceInfo(CL_DEVICE_VENDOR)`?

Reply:
hm; odd, but fair enough. no nvidia GPUs i have access to have OpenCL so I can't test that either